src/model/local_test.go

```diff
@@ -171,16 +170,16 @@ func complexExpectedConfigs() map[string]*model.Config {
 					},
 				},
 			},
-			Rows: []*model.ConfigRow{
+			Rows: []*ConfigRow{
 				{
 					BranchId:          123,
 					ComponentId:       "keboola.ex-db-mysql",
 					ConfigId:          "896",
-					Id:                "12",
-					Name:              "users",
+					Id:                "56",
+					Name:              "disabled",
```

Comment:
Toto ma trocha potrapilo. Riesit poradie config rows?

- v API je polozka `rowsSortOrder`.
- Ale v UI sa nepouziva, ... UI si radi rows interne podla `name`, na poradi v API nezalezi.
- Ak by som zacal pouzivat v `keboola-as-code` `rowsSortOrder`, tak by to bolo videt v diff, a fungovalo by to odlisne od UI
- Teda .... poradie `rows` vobec neriesime, ... kvoli testom ich zoradzujeme podla `name` ako v UI, aby nemali nahodne poradie.